cypress/e2e/fee-fine/overdue-fine-renewed-triggers.cy.js

```diff
@@ -0,0 +1,416 @@
+    });
+  };
+  const createPatronNoticeTemplate = (template) => {
+    NewNoticePolicyTemplate.startAdding();
+    NewNoticePolicyTemplate.checkInitialState();
+    NewNoticePolicyTemplate.addToken('item.title');
+    NewNoticePolicyTemplate.create(template, false);
+    NewNoticePolicyTemplate.chooseCategory(template.category);
+    NewNoticePolicyTemplate.checkPreview();
+    NewNoticePolicyTemplate.saveAndClose();
+    NewNoticePolicyTemplate.waitLoading();
+    template.category = 'AutomatedFeeFineCharge';
+    NewNoticePolicyTemplate.checkAfterSaving(template);
+  };
+  const duplicatePatronNoticeTemplate = (template) => {
+    NewNoticePolicyTemplate.duplicateTemplate();
+    NewNoticePolicyTemplate.typeTemplateName(template.name);
+    NewNoticePolicyTemplate.typeTemplateSubject(template.subject);
+    NewNoticePolicyTemplate.checkPreview();
+    NewNoticePolicyTemplate.saveAndClose();
+    NewNoticePolicyTemplate.waitLoading();
+    template.category = 'AutomatedFeeFineCharge';
+    NewNoticePolicyTemplate.checkAfterSaving(template);
+  };
+  const noticePolicy = {
+    name: `Autotest ${getRandomPostfix()} Overdue fine, returned`,
+    description: 'Created by autotest team',
+  };
+  const loanPolicyBody = {
+    id: uuid(),
+    name: `1_minute_${getRandomPostfix()}`,
+    loanable: true,
+    loansPolicy: {
+      closedLibraryDueDateManagementId: 'CURRENT_DUE_DATE_TIME',
+      period: {
+        duration: 1,
+        intervalId: 'Minutes',
+      },
+      profileId: 'Rolling',
+    },
+    renewable: true,
+    renewalsPolicy: {
+      unlimited: true,
+      renewFromId: 'CURRENT_DUE_DATE',
+    },
+  };
+  const overdueFinePolicyBody = {
+    id: uuid(),
+    name: `automationOverdueFinePolicy${getRandomPostfix()}`,
+    overdueFine: { quantity: '1.00', intervalId: 'minute' },
+    countClosed: true,
+    maxOverdueFine: '100.00',
+    forgiveOverdueFine: false,
+  };
+  const userOwnerBody = {
+    id: uuid(),
+    owner: 'AutotestOwner' + getRandomPostfix(),
+    servicePointOwner: [
+      {
+        value: testData.userServicePoint.id,
+        label: testData.userServicePoint.name,
+      },
+    ],
+  };
+
+  before('Preconditions', () => {
+    cy.getAdminToken()
+      .then(() => {
+        ServicePoints.createViaApi(testData.userServicePoint);
+        testData.defaultLocation = Location.getDefaultLocation(testData.userServicePoint.id);
+        Location.createViaApi(testData.defaultLocation);
+        cy.getInstanceTypes({ limit: 1 }).then((instanceTypes) => {
+          testData.instanceTypeId = instanceTypes[0].id;
+        });
+        cy.getHoldingTypes({ limit: 1 }).then((holdingTypes) => {
+          testData.holdingTypeId = holdingTypes[0].id;
+        });
+        cy.createLoanType({
+          name: `type_${getRandomPostfix()}`,
+        }).then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
```

Comment:
let's look for special type, for example book.

Reply:
done